types/lhr/audit-result.d.ts

```diff
@@ -76,4 +76,6 @@ export interface Result {
   };
   /** A number indicating how much guidance Lighthouse provides to solve the problem in this audit on a 1-3 scale. Higher means more guidance. */
   guidanceLevel?: number;
+  /** A list of audit ids that this audit replaces. Used to ensure the report does not render the audits in this list at the same time as the audit which contains the list. */
+  replacesAudits?: string[];
```

Comment:
This was proposed by [go/lh-rpp-roadmap](https://goto.google.com/lh-rpp-roadmap) to document the replaced audits for JSON users.

Also seems like a good way to define which old audits need to stick around in the new insight view. We could organize audits into new groups, but since this is an experimental UI phase I would rather not mess with the groups if possible.